public/js/report/controller.js

```diff
@@ -796,7 +803,6 @@ angular.module('app').controller('reportCtrl', function ($scope, connection, $co
             if (!report.properties.strokeColor) { report.properties.strokeColor = '#E0E0E0'; } // to see which ones work best for you
             if (!report.properties.generateGradient) { report.properties.generateGradient = true; }
             if (!report.properties.minValue) { report.properties.minValue = 0; }
-            if (!report.properties.maxValue) { report.properties.maxValue = 100; }
```

Comment:
It might be good to have a default value. If the text input is empty, I get an empty gauge with the text : `NaN%`